setup.py

```diff
@@ -11,7 +11,7 @@
       classifiers=['Programming Language :: Python :: 3 :: Only'],
       py_modules=['tap_outbrain'],
       install_requires=[
-          'singer-python==0.2.1',
+          'singer-python==5.6.1',
```

Comment:
Latest compatible version with current implementation, so that we can use [`singer.utils.parse_args`](https://github.com/dbt-labs/tap-outbrain/blob/7ea59804ef2a7ce10a205787c3c3a51dccbb3b78/tap_outbrain/__init__.py#L379).